lua/nvim-tree/actions/init.lua

```diff
@@ -397,6 +397,22 @@ local function copy_mappings(user_mappings)
   return user_mappings
 end
 
+local function cleanup_existing_mappings()
```

Comment:
That will remove any mappings the user manually hacked in, but their setup should add them back.